README.md

```diff
@@ -43,11 +44,75 @@ To build for Oculus Quest, you need:
 While this build process may work on Mac (untested) on the PC you gain the ability to use the Oculus Quest in `Link` mode for live in-editor debugging while wearing the headset, if your headset is set up correctly (Developer mode needs to be enabled on the device, which can only be done via the Oculus iOS / Android application for administering your device).
 
 ### Building for Hololens2
-To build for Hololens2, you need:
-* a Windows PC set up with the Windows SDK installed
-* a recent version of Visual Studio 2019 **with support for USB debugging** (an optional installation component not installed by default). Full information on Hololens build configuration [the MRTK installation guide](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Installation.html).
-
-At time of writing, this process requires MRTK 2.4, Unity 2019.4, and uses the soon-to-be-deprecated Legacy XR pipeline for Unity.
+Make sure you are using the `hl2-2019` branch when working with the HoloLens. Full information on Hololens build configuration consult [the MRTK installation guide](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Installation.html).
+
+#### Prerequisites
+The following prerequisites are required to build and deploy to a HoloLens.
```

Comment:
```suggestion
The following prerequisites are required to build and deploy to a HoloLens as of July 2021.
```
Since there are dates embedded in the versions that may change.